jaxley/solver_voltage.py

```diff
@@ -213,24 +250,22 @@ def step_voltage_implicit_with_jax_spsolve(
     n_nodes,
     internal_node_inds,
 ):
+    axial_conductances = delta_t * axial_conductances
+
     # Build diagonals.
     diagonal_values = jnp.zeros(n_nodes)
 
     # if-case needed because `.at` does not allow empty inputs, but the input is
     # empty for compartments.
     if len(sinks) > 0:
-        diagonal_values = diagonal_values.at[sinks].add(delta_t * axial_conductances)
```

Comment:
This is different from L99

Reply:
It's all good. L99 initializes with `diags = jnp.ones(nbranches * nseg)` (which corresponds to `1+...`) and here we premultiply with `delta_t` in line 253.